ckanext/visualize/templates/snippets/visualize_view_show.html

```diff
@@ -0,0 +1,78 @@
+{# Adds 'Visualize View' #}
+
+{% resource 'visualize/css/main.css' %}
+
+<!DOCTYPE html>
```

Comment:
Better to extend already existing CKAN blocks, instead of including the whole page markup here. @klikstermkd please help Miroslav with `{% super() %}` and Jinja2 block extension.